opm/autodiff/BlackoilModelParameters.cpp

```diff
@@ -64,9 +65,11 @@ namespace Opm
     void BlackoilModelParameters::reset()
     {
         // default values for the solver parameters
-        dp_max_rel_      = 1.0e9;
+        dp_max_rel_      = 1.0;
```

Comment:
Yes. Again, it is a question of robustness vs speed. I original task Ove got was to remove convergence issues not speed up the convergence.